app/src/components/HomePage.vue

```diff
@@ -136,16 +136,18 @@
       searchTerm: searchTerm || 'random'
     }))
   }
+
   const showLoader = () => {
     document.querySelector('.background-preview').style.display = 'none'
     document.querySelector('.loader').style.display = 'block'
   }
+
   const hideLoader = () => {
     document.querySelector('.background-preview').style.display = 'block'
     document.querySelector('.loader').style.display = 'none'
   }
 
-  let imageId
+  let imageId;
```

Comment:
The linter was throwing an error with this semicolon, so I pulled down your branch and removed it. But I like that you were trying to use semicolons...Vue just seems to be picky or it's with the linter on the boilerplate.